mpp/include/mpp_domains_util.inc

```diff
@@ -1693,16 +1742,29 @@ end subroutine mpp_get_tile_compute_domains
      domain_out%pe      = domain_in%pe
      domain_out%pos     = domain_in%pos
 
+     if (associated(domain_in%list)) then
```

Comment:
Is this an actual pointer, or is it something that gets allocated?

Reply:
It is a pointer that gets allocated.

Reply:
This should be changed to an allocatable then (in a different PR).